protocol/bc/hash.go

```diff
@@ -1,103 +1,92 @@
 package bc
 
 import (
-	"bytes"
 	"database/sql/driver"
-	"encoding/hex"
-	"encoding/json"
-	"fmt"
+	"encoding/binary"
 	"io"
 
 	"golang.org/x/crypto/sha3"
 
 	"chain/crypto/sha3pool"
 	"chain/encoding/blockchain"
-	"chain/errors"
 )
 
-// Hash represents a 256-bit hash.  By convention, Hash objects are
-// typically passed as values, not as pointers.
-type Hash [32]byte
+var EmptyStringHash Hash
```

Comment:
If it's not possible to initialize this in one line (`EmptyStringHash = NewHash(sha.Sum256(nil))`?) then can we have a comment here like `// initialized in func init below`?